web/src/user_group_edit.ts

```diff
@@ -314,32 +314,59 @@ function update_group_membership_button(group_id: number): void {
     }
 }
 
+function get_group_row_html(group: UserGroup): string {
+    const associated_subgroups = user_groups.get_associated_subgroups(
+        group,
+        people.my_current_user_id(),
+    );
+
+    const item: UserGroup & {
+        is_member: boolean;
+        can_join: boolean;
+        can_leave: boolean;
+        is_direct_member: boolean;
+        associated_subgroup_names: string;
+    } = {
+        is_member: user_groups.is_user_in_group(group.id, people.my_current_user_id()),
+        can_join: settings_data.can_join_user_group(group.id),
+        can_leave: settings_data.can_leave_user_group(group.id),
+        is_direct_member: user_groups.is_direct_member_of(people.my_current_user_id(), group.id),
+        associated_subgroup_names: user_groups.format_group_list(associated_subgroups),
+        ...group,
+    };
+    const html = render_browse_user_groups_list_item(item);
+    return html;
+}
+
 function update_display_checkmark_on_group_edit(group: UserGroup): void {
-    if (is_group_already_present(group)) {
-        const $row = row_for_group_id(group.id);
+    const tab_key = get_active_data().$tabs.first().attr("data-tab-key");
+    if (tab_key === "your-groups") {
+        // There is no need to do if "Your groups" tab is opened,
```

Comment:
A typo here maybe? `There is no need to do any action` or `There is no need to do anything` would be better.